javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var $background = document.querySelector('.background');
+var $bulb = document.querySelector('.light-bulb');
+var lightSwitch = 'on';
+
+function lightBulb(event) {
+  if (lightSwitch === 'on') {
```

Comment:
Instead of using a string value to compare here, try doing this with a Boolean value instead.